src/modules/simulation/gz_bridge/plugins/CMakeLists.txt

```diff
@@ -0,0 +1,49 @@
+cmake_minimum_required(VERSION 3.11.0 FATAL_ERROR)
```

Comment:
Small nitpick, but can we put the plugin directory in `simulation` and not under gz_bridge? Gz plugins should work without the `gz_bridge`

Reply:
funny you mention that because that's initially how I had it structured :joy:

Reply:
Hmm, was there someone commenting on that you should do it as it is done now?

Reply:
@dagar do you feel strongly either way?